cypress/e2e/testtab.cy.js

```diff
@@ -8,13 +8,13 @@ describe('About the Test tab', () => {
     cy.createOtherReport();
     cy.initializeApp();
     const storageIds = [];
-    cy.get('[data-cy-debug="tableBody"] tr').each($row => {
+    cy.get('[data-cy-debug="tableBody"] tbody tr').each($row => {
```

Comment:
Previously you've done: 
```ts
cy.get('[data-cy-debug="tableBody"]').get('tbody').find('tr');
```
why do it differently here, please be consistent.

Reply:
The cypress tests in general are pretty inconsistant for this. I changed it to a new command call, but it might be worth refactoring the cypress tests for this at some point.